File: 4.model_reader:shader_template:sphere/57.SphereExample3/app/src/main/java/com/example/martinrgb/a57sphereexample3/MainActivity.java
package com.example.martinrgb.a57sphereexample3;

import android.os.Bundle;
import android.app.Activity;
import android.view.Window;
import android.view.WindowManager;

public class MainActivity extends Activity {

    private OpenGLView mOpenGLView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Go to the title bar
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        //Set screen
        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN);

        mOpenGLView = new OpenGLView(this);
        setContentView(mOpenGLView);
    }
}

File: 4.model_reader:shader_template:sphere/57.SphereExample3/57.SphereExample3_kotlin/app/src/main/java/com/example/martinrgb/a57sphereexample3/MainActivity.kt
package com.example.martinrgb.a57sphereexample3

import android.os.Bundle
import android.app.Activity
import android.view.Window
import android.view.WindowManager

class MainActivity : Activity() {

    private var mOpenGLView: OpenGLView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Go to the title bar
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        //Set screen
        window.setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN, WindowManager.LayoutParams.FLAG_FULLSCREEN)

        mOpenGLView = OpenGLView(this)
        setContentView(mOpenGLView)
    }
}